roles/ohpc_add_rabbitmq_agents/tasks/main.yaml

```diff
@@ -1,41 +1,44 @@
 ---
 - name: Clone the git repo to the machine
   git: 
-    repo: https://github.com/uabrc/rabbitmq_agents.git
-    dest: /opt/rabbitmq_agents
+    repo: "{{ rabbitmq_agents_repo }}"
+    dest: "{{ rabbitmq_agents_loc }}"
     version: develop
```

Comment:
Let's go ahead an parameterize the version as well.

Reply:
we dont have release on rabbitmq agent repo yet. Should I use `rabbitmq_agents_branch` or?

Reply:
fixed in 9a07e40